src/btree/bt_vrfy.c

```diff
@@ -360,16 +367,25 @@ __verify_tree(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK *addr_unpack
 
     bm = S2BT(session)->bm;
     page = ref->page;
+    addr = NULL;
 
     unpack = &_unpack;
 
     __wt_verbose(session, WT_VERB_VERIFY, "%s %s", __wt_page_addr_string(session, ref, vs->tmp1),
       __wt_page_type_string(page->type));
 
-    /* Optionally dump the address. */
-    if (vs->dump_address)
-        WT_RET(__wt_msg(session, "%s %s", __wt_page_addr_string(session, ref, vs->tmp1),
-          __wt_page_type_string(page->type)));
+    /* Optionally dump the address/timestamp information. */
+    if (vs->dump_address || vs->dump_time_pairs) {
+        WT_RET(__wt_msg(session, "%s", __wt_page_type_string(page->type)));
+        if (vs->dump_address)
+            WT_RET(__wt_msg(session, "\t%s", __wt_page_addr_string(session, ref, vs->tmp1)));
+        if (vs->dump_time_pairs) {
+            addr = ref->addr;
+            if (addr != NULL)
+                WT_RET(__wt_msg(session, "\t<%lx:%lx, %lx:%lx>", addr->oldest_start_ts,
```

Comment:
@sulabhM you may be interested in this. We're printing timestamps as Hex which I think makes more sense.

Reply:
@luke-pearson This isn't aimed at me - I think it's important that we print timestamps in the same format that they are printed from MongoDB. Let me know if you need help finding an example of how that is done.

Reply:
I second @agorrod , it will be best to use ```__wt_timestamp_to_string()``` which prints in MongoDB ways.

Reply:
In keeping with the current debug printing format in `__debug_cell`, I've changed it to ts/txn,ts/txn